docs/contribute/locally.md

```diff
@@ -96,6 +96,10 @@ This guide uses option one. If you'd like to clone the repository and build from
 
 ::::
 
+:::{tip}
+Place the `docs-builder` binary file in a directory and add its path to the system paths to run it from any folder. On macOS, edit the paths file by running `sudo nano /etc/paths`.
```

Comment:
What do you think about moving the binary to `/usr/local/bin` ?

Reply:
It's what I've done locally, but I wonder if tampering with that folder wouldn't be frowned upon by, say, IT. :-S

Reply:
Good question. I'm putting all my binaries to that folder 🤷.

I did some quick Google search and storing a custom binary in `/usr/local/bin` seems to be the most common suggestion.

Reply:
Let's do that. I'll edit the tip with a sudo mv command.

Reply:
Have a look now!

Reply:
Cool. Thank you for the contribution!